### PR TITLE
Adafruit Gemma support

### PR DESCRIPTION
**Pull request by bkudria**

### CODE REVIEW

File: platformio/boards/adafruit.json [hunk 1]
```diff
@@ -47,6 +47,26 @@
         "vendor": "Adafruit"
     },
 
+    "gemma": {
+        "build": {
+            "core": "arduino",
+            "extra_flags": "-DARDUINO_ARCH_AVR -DARDUINO=10605 -DARDUINO_AVR_GEMMA",
```
Comment (ivankravets) [inline platformio/boards/adafruit.json]: Please remove `-DARDUINO=10605`. PlatformIO will add it later automatically.

**Comment by ivankravets** [comment]
@bkudria Could I ask you to look in my comment for pull-request?

**Comment by bkudria** [comment]
@ivankravets sorry, I pushed an updated commit, but I forgot to ping you! The flags should be as you requested!

**Comment by ivankravets** [comment]
Thanks a lot!

**Pull request merged**